src/main/java/com/codessquad/qna/domain/Question.java

```diff
@@ -91,16 +98,32 @@ public void setAnswers(List<Answer> answers) {
         this.answers = answers;
     }
 
-    public int getAnswerCount() {
-        return answers.size();
-    }
-
     public boolean matchUser(User loginUser) {
         String userId = loginUser.getUserId();
-        if (this.writer.getUserId().equals(userId)) {
+        return this.writer.getUserId().equals(userId);
+    }
+
+    public void delete() {
+        this.isDelete = true;
+    }
+
+    public boolean isDelete() {
+        return isDelete;
+    }
+
+    public boolean canDelete(Question question, User loginUser, List<Answer> activeAnswers) {
+        if (!question.matchUser(loginUser)) {
```

Comment:
```suggestion
        if (!question.isMatch(loginUser)) {
```

정도로 boolean 타입 값을 리턴하는 느낌을 줄 수 있을 것 같네요.